lib/components/map/base-map.js

```diff
@@ -125,7 +76,8 @@ const mapStateToProps = (state, ownProps) => {
 
 const mapDispatchToProps = (dispatch, ownProps) => {
   return {
-    setLocation: (type, location) => { dispatch(setLocation({ type, location })) }
+    setLocation: (type, location) => { dispatch(setLocation({ type, location })) },
+    updateMapState: (props) => { dispatch(updateMapState(props)) }
```

Comment:
Thanks for the note @trevorgerhardt.  I didn't realize it could be reduced to that.